package.json

```diff
@@ -30,7 +30,9 @@
     "dotenv": "^4.0.0",
     "json-loader": "latest",
     "jsonfile": "^3.0.1",
+    "mem": "^4.0.0",
     "mysql": "^2.14.1",
-    "req": "^0.1.4"
+    "req": "^0.1.4",
+    "stringstream": "0.0.6"
```

Comment:
I'm not too familiar with yarn, but I still don't think that this is necessary. If this dependency is removed, the `0.0.6` version upgrade in `yarn.lock` still persists for `stringstream@~0.0.4` after running `yarn install`.

Reply:
I can't update the breaking packages without doing this. I used yarn upgrade and this was added